telegraf/plan.sh

```diff
@@ -0,0 +1,47 @@
+pkg_name=telegraf
+pkg_origin=core
+pkg_version="1.3.5"
+pkg_license=('MIT')
+pkg_description="telegraf - client for InfluxDB"
+pkg_upstream_url="https://github.com/influxdata/telegraf/"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_scaffolding="core/scaffolding-go"
```

Comment:
It feels like we're using the go scaffolding here for ceremony. Could we use the [releases](https://github.com/influxdata/telegraf/releases) from the upstream? I think this would give us shasum verification and tidy up this plan quite a bit